sa/database.go

```diff
@@ -156,12 +160,103 @@ func NewDbMapFromConfig(config *mysql.Config, settings DbSettings) (*boulderDB.W
 	dbmap := &gorp.DbMap{Db: db, Dialect: dialect, TypeConverter: BoulderTypeConverter{}}
 
 	initTables(dbmap)
-
 	return &boulderDB.WrappedMap{DbMap: dbmap}, nil
 }
 
-// adjustMySQLConfig sets certain flags that we want on every connection.
-func adjustMySQLConfig(conf *mysql.Config) {
+var checkQuoteRE = regexp.MustCompile(`^'{1}\w+'{1}$`)
```

Comment:
Are we guaranteed that all mariaDB system variable values will match this regex? Is it never permissible for a string variable to have a comma, space, semicolon, etc in it?